app/src/components/CalibrateLabware/ProceedToRun.js

```diff
@@ -11,6 +11,7 @@ import type { Dispatch } from '../../types'
 import pcrSealSrc from '../../img/place_pcr_seal.png'
 import { Portal } from '../portal'
 import styles from './styles.css'
+import { THERMOCYCLER } from '@opentrons/shared-data/js/constants'
```

Comment:
Constants are re-exported from the `main`, so this should be:

```suggestion
import { THERMOCYCLER } from '@opentrons/shared-data'
```